doc/flame/game.md

```diff
@@ -87,6 +87,24 @@ if you change the `debugMode` at runtime, it will not affect already added compo
 To read more about the `debugMode` on Flame, please refer to the [Debug Docs](other/debug.md)
 
 
+# Change background color
+
+To change the background color of your `FlameGame` you have to override `backgroundColor()`.
+
+In the following example the background color is set to be fully transparent, so that you can see
+the widgets that are behind the `GameWidget`. The default it opaque black.
+
+```dart
+class MyGame extends FlameGame {
+  @override
+  Color backgroundColor() => const Color(0x00000000);
+}
+```
+
+Note that the background color can't change dynamically while the game is running, but you could
```

Comment:
This seems like an artificial limitation. Should we convert this into a settable property and allow to change the background color at runtime?

Reply:
If you can find a way, go for it, but it's not as simple as it may sound since it's done in the widget. I don't think we should start drawing the background manually.